contracts/libs/FixedPointInt256.sol

```diff
@@ -0,0 +1,101 @@
+/**
+ * SPDX-License-Identifier: UNLICENSED
+ */
+pragma solidity 0.6.10;
+
+import "../packages/oz/SignedSafeMath.sol";
+
+/**
+ *
+ */
+library FixedPointInt256 {
+    using SignedSafeMath for int256;
+
+    int256 private constant SCALING_FACTOR = 1e18;
+
+    /**
+     * @notice convert an unsigned integer to signed integer
+     * @param a uint to convert into a signed integer.
+     * @return the converted signed integer.
+     */
+    function fromUint(uint256 a) internal pure returns (int256) {
+        require(a < uint256(-1), "FixedPointInt256: out of int range");
+
+        return int256(a);
+    }
+
+    /**
+     * @notice convert a signed integer to unsigned integer
+     * @param a int to convert into an unsigned integer.
+     * @return the converted unsigned integer.
+     */
+    function fromInt(int256 a) internal pure returns (uint256) {
+        if (a < 0) {
+            return uint256(-a);
+        } else {
+            return uint256(a);
```

Comment:
does this need an overflow / underflow check?

Reply:
I don't think so, it is meant to solve this issue https://solidity.readthedocs.io/en/latest/types.html#explicit-conversions